desktop_version/src/scripts/terminal_finallevel.vsc

```diff
@@ -0,0 +1,69 @@
+cutscene()
+untilbars()
+
+squeak(terminal)
+text(gray,0,114,9)
+* DIMENSIONAL STABILITY GENERATOR *
+
+     [ Currently Generating ]      
+Maximum Stability    
+
+    [ Status ]
+      Online
+
+READY _
+position(center)
+speak_active
+
+endtextfast
+delay(10)
+
+squeak(player)
+text(cyan,0,0,2)
+Aha! This must be what's ,
```

Comment:
There's a comma at the end of this line (context from the original hardcoded script to see what went wrong:)
```cpp
        "text(cyan,0,0,2)",
        "Aha! This must be what's" ,
        "causing the interference!",
```

Reply:
Good catch! I'll update my scripts for extracting these to their own files and update the branch.